scripts/insert_into_appointments.sql

```diff
@@ -1 +1,7 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(patient_name, doctor_name, appointment_time, appointment_date, appointment_comment,doctor_id, patient_id)
```

Comment:
Does the `appointments` table have columns called `patient_name` or `doctor_name`?